nexus/src/authn/external/session_cookie.rs

```diff
@@ -55,8 +55,10 @@ pub const SESSION_COOKIE_SCHEME_NAME: authn::SchemeName =
 
 /// Generate session cookie header
 pub fn session_cookie_header_value(token: &str, max_age: Duration) -> String {
+    // TODO(https://github.com/oxidecomputer/omicron/issues/249): We should
+    // insert "Secure;" back into this string.
```

Comment:
Dave has been using `TODO-security` to make these greppable all at once.

Reply:
Done